Directory.Build.props

```diff
@@ -6,6 +6,7 @@
     <IsTestProject Condition="$(MSBuildProjectName.EndsWith('.Tests'))">true</IsTestProject>
     <CentralPackagesFile>$(MSBuildThisFileDirectory)Packages.Data.props</CentralPackagesFile>
     <AzureCoreSharedSources>$(MSBuildThisFileDirectory)/src/assets/Azure.Core.Shared/</AzureCoreSharedSources>
+    <FakeAzureCoreTestFrameworkSources>$(MSBuildThisFileDirectory)/src/assets/TestFramework/</FakeAzureCoreTestFrameworkSources>
```

Comment:
Can we just name `AzureCoreTestFrameworkSources`?

Reply:
Because they are fake, they are only here to make sure our code could compile.
TestFramework is not a library, we cannot depend on them, therefore we create files to mimic their interfaces.